### ISSUE TITLE
RegressionHead, how to set output layer activation function

### ISSUE THREAD

**Issue by Astlaan** [keras-team/autokeras#1165]
Is there a way to set the activation function for the output layer of a regression head?
I need to set it to sigmoid (but it's a regression problem, so I don't intend to use ClassificationHead).

**Comment by Astlaan** [comment]
I was able to get around this issue by overriding RegressionHead.build, by creating a `RegressionHeadSigmoid` that inherits from it and actually passes the activation parameter to the Dense layer.

```
class RegressionHeadSigmoid(ak.RegressionHead):
    def build(self, hp, inputs=None):
        if self.output_dim and self.output_shape[-1] != self.output_dim:
            raise ValueError(
                'The data doesn\'t match the output_dim. '
                'Expecting {} but got {}'.format(self.output_dim,
                                                 self.output_shape[-1]))
        inputs = tf.nest.flatten(inputs)
        ak.utils.validate_num_inputs(inputs, 1)
        input_node = inputs[0]
        output_node = input_node

        dropout_rate = self.dropout_rate or hp.Choice('dropout_rate',
                                                      [0.0, 0.25, 0.5],
                                                      default=0)

        if dropout_rate > 0:
            output_node = layers.Dropout(dropout_rate)(output_node)
        output_node = ak.Flatten().build(hp, output_node)
        output_node = tf.keras.layers.Dense(self.output_shape[-1],
                                   name=self.name, activation='sigmoid')(output_node)
        return output_node
```

However, I think it would be nice to modify RegressionHead to allow this 'activation' parameter as part of its constructor.

**Comment by d-chambers** [comment]
I am having this same issue. I want to force the output of the model to be between 0 and 1 so a dense output layer with a sigmoid activation seems appropriate. However, I am not able to use the class defined here as output in an `AutoModel`, it just keeps raising a `NotImplementedError` and fails to build the model.
